resources/js/cover.js

```diff
@@ -0,0 +1,15 @@
+const dummyCover = 'resources/images/dummyimage1.jpg';
+const str = '#BLOG';
+
+let coverBackground = document.getElementById('cover');
```

Comment:
why using let ? :v

Reply:
Because of Duy's advices and as far as i khow, 'var' is function scoped while 'let' is block scoped

Reply:
We should use const here. Additionally, coverBlock is a better name instead of coverBackground

Reply:
@ngoc-nguyenminh fix này để a còn merge e

Reply:
@MiloNguyen95  dạ e fix rồi đó a, sorry anh e tối qua ngủ sớnm quá :(((